SDL/FxCopSetup.ps1

```diff
@@ -7,13 +7,33 @@ Param(
     [string]$fxCopDirectory = (Join-Path -Path (Split-Path -parent (Split-Path -parent (Split-Path -parent $PSCommandPath))) -ChildPath "fxCop")
 )
 
+function IsFileDependency {
+    [CmdletBinding()]
+    param
+    (    
+    $file
+    )
+        $dependencyFiles | ForEach-Object {
+            if($file.Name -eq $_) {
+                return $true;
+            } 
+        }
+    
+        return $false;
+    }
+
 # these are dlls that end up in the bin, but do not belong to us and don't need to be scanned.
 $excludedFiles = @();
+$dependencyFiles = @("Microsoft.ApplicationInsights.dll", "Microsoft.AspNet.TelemetryCorrelation.dll", "Microsoft.Web.Infrastructure.dll", "System.Diagnostics.DiagnosticSource.dll");
```

Comment:
"Microsoft.ApplicationInsights.dll" - - this is our own file right? or is it covered in base sdk build , and hence not required here?

Reply:
yeah, that one is covered in the Base SDK repo :)